lib/extractor.rb

```diff
@@ -142,6 +143,15 @@ def get_extract(filepath)
       type_elem.value = "preview"
     end
 
+    unless @title_prefix.nil?
+      if @source_book.property_for("title").nil?
+        @source_book.add_property("title", @title_prefix)
+      else
+        title_elem = @source_book.properties.select{|p| p.key == "title"}.first
+        title_elem.value = @title_prefix + ' ' + title_elem.value
```

Comment:
>  Prefer string interpolation and string formatting instead of string concatenation
https://github.com/bbatsov/ruby-style-guide#string-interpolation